### ISSUE TITLE
compile error on execute without query parameters

### ISSUE THREAD

**Issue by sabelka** [dmitigr/pgfe#9]
I get a compiler error when I try to compile an database query without any bind argument, like the following:

```
#include "dmitigr/pgfe.hpp"

using namespace dmitigr::pgfe;

void foo(Connection& conn)
{
  conn.execute("begin");
}
```

This is the error I get:
```
In file included from /home/test/lib/pgfe/lib/dmitigr/pgfe/connection.hpp:17,
                 from /home/test/lib/pgfe/lib/dmitigr/pgfe/pgfe.hpp:32,
                 from /home/test/lib/pgfe/lib/dmitigr/pgfe.hpp:30,
                 from /home/test/src/test.cpp:1:
/home/test/lib/pgfe/lib/dmitigr/pgfe/prepared_statement.hpp: In instantiation of ‘dmitigr::pgfe::Prepared_statement& dmitigr::pgfe::Prepared_statement::bind_many__(std::index_sequence<I ...>, Types&& ...) [with long unsigned int ...I = {}; Types = {}; std::index_sequence<I ...> = std::integer_sequence<long unsigned int>]’:
/home/test/lib/pgfe/lib/dmitigr/pgfe/prepared_statement.hpp:441:23:   required from ‘dmitigr::pgfe::Prepared_statement& dmitigr::pgfe::Prepared_statement::bind_many(Types&& ...) [with Types = {}]’
/home/test/lib/pgfe/lib/dmitigr/pgfe/connection.hpp:860:18:   required from ‘std::enable_if_t<dmitigr::pgfe::detail::Response_callback_traits<F>::is_valid, dmitigr::pgfe::Completion> dmitigr::pgfe::Connection::execute(F&&, const dmitigr::pgfe::Sql_string&, Types&& ...) [with F = dmitigr::pgfe::Connection::execute<{}>::<lambda(auto:27&&)>; Types = {}; std::enable_if_t<dmitigr::pgfe::detail::Response_callback_traits<F>::is_valid, dmitigr::pgfe::Completion> = dmitigr::pgfe::Completion]’
/home/test/lib/pgfe/lib/dmitigr/pgfe/connection.hpp:868:19:   required from ‘dmitigr::pgfe::Completion dmitigr::pgfe::Connection::execute(const dmitigr::pgfe::Sql_string&, Types&& ...) [with Types = {}]’
/home/test/src/test.cpp:8:23:   required from here
/home/test/lib/pgfe/lib/dmitigr/pgfe/prepared_statement.hpp:676:54: error: invalid initialization of non-const reference of type ‘dmitigr::pgfe::Prepared_statement&’ from an rvalue of type ‘void’
  676 |     return (bind__(I, std::forward<Types>(args)), ...);
```

However, when I add some bind argument, like `conn.execute("begin", 1);`, then it compiles file without any errors!

**Comment by dmitigr** [comment]
This is fixed in just pushed v2.0alpha2. Thank you for the report!

**Comment by sabelka** [comment]
Just pulled and everything works! Thank you!
